index.bs

```diff
@@ -805,6 +805,57 @@ and doing cleanup as necessary (see above).
   is still [under discussion](https://github.com/whatwg/html/issues/6588).
 </div>
 
+<h3 id="third-party-tools">Prioritize usability over compatibility with third-party tools</h3>
+
```

Comment:
@cyntha How about this for the tl;dr?

```suggestion
New web platform features should be designed with usability as the primary goal, and compatibility with third party tools as a secondary one.
They should never however break existing web content or existing web platform features with new ones.

```

Reply:
Suggestion for 809: Design new features with usability as the primary goal, and compatibility with third-party tooling as a secondary goal.